Core/include/Acts/Utilities/Grid.hpp

```diff
@@ -196,6 +263,12 @@ class Grid final : public IGrid {
     return m_values.at(globalBinFromLocalBins(localBins));
   }
 
+  /// @copydoc IGrid::atLocalBinsAny(AnyIndexType)
+  std::any atLocalBinsAny(AnyIndexType indices) const override {
+    const_reference cref = atLocalBins(toIndexType(indices));
+    return &cref;
```

Comment:
I see, this disallows `bool` as entry, but that's ok, don't think we need a grid of bools.